### DISCUSSION TITLE
有人可以幫我解惑嗎

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#35] [Q&A]
![python 問題](https://user-images.githubusercontent.com/91265127/136365249-1247b8d2-a13e-41b9-b999-825ee4961ba4.PNG)

請問要怎麼修改讓這段程式碼運作?
抱歉!我是萌新

**Comment by jacky0115** [comment]
![image](https://user-images.githubusercontent.com/86584463/136370162-e1dc1f83-e669-49a3-8d64-d9518750e442.png)
str用法 : **str(object=b'', encoding='utf-8', errors='strict')**
str裡面放的第一個參數是物件，裡面不能放賦值語句
或者你也可以選擇直接印出來
